cypress/e2e/awx/resources/workflowJobTemplates.cy.ts

```diff
@@ -208,17 +415,62 @@ describe('Workflow Job Templates Tests', () => {
       cy.deleteAwxOrganization(organization, { failOnStatusCode: false });
     });
 
-    it.skip('can delete a workflow job template from the details page', () => {
-      //Utilize a workflow job template created in the beforeEach block
-      //Assert original values on the WFJT
-      //After delete, intercept the API response and assert the deletion
+    it('can delete a workflow job template from the details page', () => {
```

Comment:
the `describe` block here is creating `organization`, `inventory` and `workflow template` for each test but in 2 out of 3 `it` tests blocks `organization` and `inventory` are not used. We are trying to avoid creating unnecessary resources if they are not being used in the test. For those delete tests i would suggest to add them in a different `describe` block.